server/boot/user.js

```diff
@@ -266,6 +272,42 @@ module.exports = function(app) {
     return res.redirect('/' + username);
   }
 
+  function getUnlinkSocial(req, res, next) {
```

Comment:
This method does not delete the userIdentitty models associated with this account